docs/troubleshooting.asciidoc

```diff
@@ -49,15 +49,27 @@ kibana-sample-kb-http          ClusterIP   10.19.246.116   <none>        5601/TC
 [id="{p}-describe-failing-resources"]
 === Describe failing resources
 
-If an Elasticsearch node does not come up, it's probably because Kubernetes cannot schedule the associated Pod. Check the Pod status: if it's not `Running` after a few seconds, something is wrong:
+If an Elasticsearch node does not come up, it is probably because Kubernetes cannot schedule the associated Pod.
+
+Start by looking at the StatefulSets to check that the current number of replicas match the desired number of replicas.
 
 [source,sh]
 ----
-kubectl get pods
+kubectl get statefulset
+
+NAME                              DESIRED   CURRENT   AGE
+elasticsearch-sample-es-default   1         1         4s
+----
+
+Next, check the Pod status. If a pod fails to reach the `Running` status after a few seconds, something could be preventing it from being scheduled or starting up:
```

Comment:
Then, check the Pod status. If a Pod fails to reach the `Running` status after a few seconds, something is preventing it from being scheduled or starting up: